n26/api.py

```diff
@@ -295,6 +304,84 @@ def _do_request(self, method: str = GET, url: str = "/", params: dict = None,
         if len(response.content) > 0:
             return response.json()
 
+    def get_encryption_key(self, public_key: str = None) -> dict:
+        """
+        Receive public encryption key for the JSON containing the encrypted PIN
+        """
+        return self._do_request(GET, BASE_URL_DE + '/api/encryption/key', params={
+            'publicKey': public_key})
+
+    def encrypt_user_pin(self, pin: str):
+        """
+        Encrypts user PIN and prepares to send it in a format specified in the documentation
+
+        :return: base64 encoded JSON containing the encrypted PIN and its encryption key
+        """
+        # generate AES256 key and IV
+        random_password = Random.get_random_bytes(32)
+        salt = Random.get_random_bytes(16)
+        key = PBKDF2(random_password, salt, 32, count=1000000, hmac_hash_module=SHA512)
```

Comment:
When checking out your repository I noticed that pylint complains about the types passed into this constructor.
Since it seems to work just fine please add a `# noinspection` line like this:
```
# noinspection PyTypeChecker
key = PBKDF2(random_password, salt, 32, count=1000000, hmac_hash_module=SHA512)
```